### PR TITLE
games-fps/urbanterror: Bump to 4.3.2_p20171016.

### PR DESCRIPTION
**Pull request by holgersson32644**
Package-Manager: Portage-2.3.12, Repoman-2.3.3

### CODE REVIEW

File: games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+		if ! use openal && ! use opus && ! use vorbis; then
+			ewarn
+			ewarn "No sound implementation selected. Enable 'openal', 'opus' or 'vorbis' USE flag to get sound!"
+		fi
+	fi
+}
+
+src_compile() {
+	buildit() { use $1 && echo 1 || echo 0 ; }
+	nobuildit() { use $1 && echo 0 || echo 1 ; }
```
Comment (candrews) [inline games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild]: Don't use `buildit` and `nobuildit` - see https://devmanual.gentoo.org/ebuild-writing/functions/src_configure/configuring/index.html For example, consider `usex` instead

File: games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+	# Workaround for used zlib macro, wrt bug #44951
```
Comment (candrews) [inline games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild]: https://bugs.gentoo.org/44951 doesn't seem related. Is this the right reference?

File: games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+		USE_CODEC_OPUS=$(buildit opus) \
+		USE_MUMBLE=$(buildit mumble) \
+		USE_VOIP=$(buildit voip) \
+		USE_INTERNAL_LIBS=0 \
+		USE_LOCAL_HEADERS=0 \
+		USE_ALTGAMMA=$(buildit altgamma)
+}
+
+src_install() {
+	local my_arch=$(usex amd64 "x86_64" "i386")
```
Comment (candrews) [inline games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild]: Define `my_arch` globally and use it in both `src_install` and at https://github.com/gentoo/gentoo/pull/6050/files#diff-41ab06224e69e322727523fb726ab7e0R65

File: games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild (hunk 1 above)
Comment (holgersson32644) [reply games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild]: fixed.

File: games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild (hunk 2 above)
Comment (holgersson32644) [reply games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild]: I think it was https://bugs.gentoo.org/449510 - and I dropped the trailing zero somewhere.
Fixed.

File: games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild (hunk 3 above)
Comment (holgersson32644) [reply games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild]: This doesn’t work in global scope.

Comment (candrews) [reply games-fps/urbanterror/urbanterror-4.3.2_p20171016.ebuild]: You're right, of course

**Comment by candrews** [comment]
a5aba94830934c4703d8526f32078c5f6bbda122
